src/test/java/net/openhft/chronicle/salt/SHA2Test.java

```diff
@@ -31,7 +31,9 @@ private static void doTest256(byte[] inputStr, String expectedHex) {
     }
 
     private static void doTest512(String inputStr, String expectedHex) {
-        Bytes<?> input = Bytes.allocateElasticDirect();
+        Bytes<?> input = Bytes.allocateElasticDirect()
+                // Forces replacement of the underlying BytesStore for ""
+                .write(new byte[0]);
```

Comment:
Why? Is there a chance our users will also have to do this? Does allocateElasticDirect() return an EmptyByteStore with all of its methods marked as Unsupported?

Reply:
The problem is specifically for low-level methods like addressForRead. They should not be used by end-users.

Reply:
`allocateElasticDirect()` returns a `Bytes` initially backed by the `EmptyByteStore`.